docs/README.md

```diff
@@ -1,6 +1,7 @@
 # php-src docs
 
-This is the home of the php-src internal documentation. It is in very early stages, but is intended
+This is the home of the php-src internal documentation, hosted at
+[php.github.io/php-src/](https://php.github.io/php-src/). It is in very early stages, but is intended
```

Comment:
Sorry to be pedantic, these are 101 characters :slightly_smiling_face:

Reply:
there were another line over 100 chars

I wonder a little why 100 is the limit instead of typical 120 chars

Reply:
No special reason, just personal preference. I find it better for readability than 120 while not as extreme as 80.